controller/model/enrollment_mod_ott.go

```diff
@@ -99,11 +100,15 @@ func (module *EnrollModuleOtt) Process(ctx EnrollmentContext) (*EnrollmentResult
 		return nil, err
 	}
 
+	content := &rest_model.EnrollmentCerts{
+		Cert: string(certPem),
+	}
+
 	return &EnrollmentResult{
 		Identity:      identity,
 		Authenticator: newAuthenticator,
-		Content:       certPem,
-		ContentType:   "application/x-pem-file",
```

Comment:
hrmmm. I'm thinking the ContentType removal was purposeful now. I don't expect anything relied on the ContentType and it's perfectly fine to remove it

Reply:
Sure was see your other comment!